plugin/trino-hive/src/main/java/io/trino/plugin/hive/util/HiveTypeTranslator.java

```diff
@@ -144,6 +147,9 @@ public static TypeInfo toTypeInfo(Type type)
         if (type instanceof TimestampType) {
             return HIVE_TIMESTAMP.getTypeInfo();
         }
+        if (type instanceof TimestampWithTimeZoneType) {
+            return HIVE_TIMESTAMPLOCALTZ.getTypeInfo();
+        }
```

Comment:
I'm not sure this is the right choice. If we want to add support for Hive's TIMESTAMP WITH TIME ZONE type later, this will be unexpected.

Reply:
I chose to go with TIMESTAMP WITH LOCAL TIME ZONE, because it's roughly Hive equivalent of Instant. And in practice, the formats used with Hive (ORC, Parquet, Avro) store the timestamps as instants (they don't keep timezone information with each value). 

I'm not sure if there is any way to to support both TIMESTAMP WITH TIME ZONE and TIMESTAMP WITH LOCAL TIME ZONE (at least without some custom annotations), considering Trino doesn't have a separate Instant-like type.

Reply:
There are a few things to consider:
* How does a Hive `TIMESTAMP WITH LOCAL TIME ZONE` present within Trino? We concluded that it'd be reasonable to expose that as a Trino `TIMESTAMP WITH TIME ZONE` with an arbitrary time zone.
* How does a Hive `TIMESTAMP WITH TIME ZONE` present within Trino? We don't have an answer yet, and it's out of the scope of this change. The likely reasonable choice is to expose it as a Trino `TIMESTAMP WITH TIME ZONE`.
* How does a Trino `TIMESTAMP WITH TIME ZONE` present within Hive? Specifically, when creating a table. This is also outside the scope of this PR, but the likely choice is also a Hive `TIMESTAMP WITH TIME ZONE`.

Reply:
Oh, I see - that makes sense. I'll remove this line and adjust the tests.

Reply:
Done - I removed mapping from Trino to Hive types (and had to change test to create the table directly in the metastore)